### DISCUSSION TITLE
How to run the C# interactive mode?

### DISCUSSION THREAD

**Discussion by paulomorgado** [badamczewski/PowerUp#15] [Q&A]
[READEME.md](https://github.com/badamczewski/PowerUp/blob/main/README.md#c-decompilation-and-dissasembly) states that:

> C# is the only language that supports (at the moment) running code, interactive mode, benchmarking and class and struct layouts.

But I can't find out how.

**Comment by badamczewski** [comment]
You have to use a code editor like VS code then open the C# file along with the X86 ASM file, change the C# file and the watcher will update the ASM file, which will reload in VS Code :)
